lab_2/main.py

```diff
@@ -57,19 +55,18 @@ def fill_lcs_matrix(first_sentence_tokens: tuple, second_sentence_tokens: tuple)
         return []
     if len(first_sentence_tokens) == 0 or len(second_sentence_tokens) == 0:
         return []
-    for word in first_sentence_tokens:
-        if not isinstance (word, str):
-            return []
-    for word in second_sentence_tokens:
-        if not isinstance (word, str):
-            return []
+    list_of_tuples = [first_sentence_tokens, second_sentence_tokens]
+    for current_tuple in list_of_tuples:
+        for word in current_tuple:
+            if not isinstance(word, str):
+                return []
     matrix = create_zero_matrix(len(first_sentence_tokens) + 1, len(second_sentence_tokens) + 1)
     new_matrix = create_zero_matrix(len(first_sentence_tokens), len(second_sentence_tokens))
```

Comment:
imagine that `matrix` is large, do you want to create another large matrix?
think of a way how not to store two matrices